include/IEntity.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+** EPITECH PROJECT, 2023
+** Visual Studio Live Share (Workspace)
+** File description:
+** IEntity
+*/
+
+#pragma once
+
+#include <vector>
+#include "IComponent.hpp"
+
+namespace Arcade {
+    namespace ECS {
+        class IEntity {
```

Comment:
No doc before IEntity ? To explain what it is

Reply:
It's an ECS, anyone who will use this, I hope understood it before the usage of this interface (at meetings that we set-up every day)